plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/debug/AbstractDebugger.java

```diff
@@ -710,15 +701,10 @@ private VariablePathDto asDto(VariablePath variablePath) {
     }
 
     /**
-     * Transforms FQN to file path.
-     */
-    abstract protected List<String> fqnToPath(@NotNull Location location);
-
-    /**
      * Transforms file path to FQN>
      */
     @Nullable
-    abstract protected String pathToFqn(VirtualFile file);
+    abstract protected String pathToFqn(VirtualFile file);//strange name for generic debugger
```

Comment:
This abstract class we use for creating implementation not only java debugger. I think word "fqn" is popular in the Java language, but not for C, C++.